src/schema/v2/artwork/index.ts

```diff
@@ -727,18 +730,20 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
             ? "within Continental Europe"
             : "domestic"
 
-          if (
-            domesticShipping &&
-            artwork.international_shipping_fee_cents == null
-          )
-            return `Shipping: ${domesticShipping} ${shippingRegion} only`
-
           if (artwork.domestic_shipping_fee_cents === 0)
             domesticShipping = "Free"
           if (artwork.international_shipping_fee_cents === 0)
             internationalShipping = "free"
 
-          return `Shipping: ${domesticShipping} ${shippingRegion}, ${internationalShipping} rest of world`
+          const domesticShippingMessage = artwork.arta_enabled
+            ? "domestic calculated at checkout"
+            : `${domesticShipping} ${shippingRegion}`
+
+          if (artwork.international_shipping_fee_cents == null) {
+            return `Shipping: ${domesticShippingMessage} only`
+          }
+
+          return `Shipping: ${domesticShippingMessage}, ${internationalShipping} rest of world`
```

Comment:
non-blocking: 👍 I found this function a bit hard to read, I'll open a ticket to try to refactor this function to be a bit more straightforward!